### PR TITLE
add new error message

### PR DESCRIPTION
**Pull request by Tushu17**
### Details

Added new error message

### Fixed Issues

$ https://github.com/Expensify/App/issues/7269

### Tests

- [ ] Verify that no errors appear in the JS console

### QA Steps

1. Navigate to Workspace > Add bank account or Settings > Payment > Add Payment > Add bank account
2. Select Plaid > Wells Fargo & login
3. Select Savings account and enter incorrect account password
4. Verify that the error message says "Incorrect password. Please try again." or "Contraseña incorrecta. Por favor inténtalo de nuevo"
- [ ] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

### CODE REVIEW

**Comment by Tushu17** [comment]
I haven't attached any screenshots as it can't be tested on development. It shows CORS error in local host.

**Comment by Tushu17** [comment]
**_I have read the CLA Document and I hereby sign the CLA_**

**Comment by Tushu17** [comment]
recheck

File: src/languages/es.js [hunk 1]
```diff
@@ -430,6 +430,7 @@ export default {
         twoFactorCode: 'Autenticación de 2 factores',
         requiredWhen2FAEnabled: 'Obligatorio cuando A2F está habilitado',
         error: {
+            incorrectPassword: 'Contraseña incorrecta. Por favor inténtalo de nuevo',
```
Comment (parasharrajat) [inline src/languages/es.js:433]: Is there supposed to be a full stop at the end? I think @pecanoro can confirm this.

**Review by parasharrajat — APPROVED** [review]
LGTM. It is a small change and I can confirm it will work so it's fine to skip the screenshots.
cc: @pecanoro I have a small doubt for you.

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by parasharrajat — CHANGES_REQUESTED** [review]
@Tushu17 Please sign your commits.

File: src/languages/es.js (hunk 1 above)
Comment (pecanoro) [reply src/languages/es.js:433]: Yes, I would say so since the English version has the full stop at the end.

**Review by parasharrajat — CHANGES_REQUESTED** [review]
You would need to sign all of your commits before this can be merged.

**Comment by Tushu17** [comment]
@parasharrajat, is it good now?

**Review by parasharrajat — APPROVED** [review]

**Review by pecanoro — APPROVED** [review]

**Comment by mvtglobally** [comment]
@Tushu17 @pecanoro Should it be QAed on all platforms? Checkboxes are missing

**Comment by Tushu17** [comment]
@mvtglobally Yeah it's for all platforms.

**Comment by mvtglobally** [comment]
After few PRs and CPs were deployed this round, we are no longer able to se password field on Desktop. Other platforms were pass 
Screen Shot 2022-01-28 at 1 24 01 PM

**Comment by mvtglobally** [comment]
Ignore previous message. Desktop is pass too

**Pull request merged**